index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
WHy this file in the root folder? The one in the `public` is not used?

Reply:
Good point, I will try to use that one!

Reply:
After checking the docs, `index.html` for vite project needs to be in project root: https://vitejs.dev/guide/#index-html-and-project-root

I removed the public/index.html to avoid confusion